backend/modules/query_controllers/default/controller.py

```diff
@@ -37,14 +38,14 @@ async def answer(self, request: DefaultQueryInput):
 
             vector_store = VECTOR_STORE_CLIENT.get_vector_store(
                 collection_name=collection.name,
-                embeddings=get_embedder(collection.embedder_config),
+                embeddings=OpenAIEmbeddings(
```

Comment:
use get_embedder function only and instead of truefoundry embedding use this class